### DISCUSSION TITLE
GitHub Copilot

### DISCUSSION THREAD

**Discussion by gnl42** [eclipse-mylyn/org.eclipse.mylyn#424] [General]
Just wondering about GitHub Copilot status? 

I tried to activate it but I was given a activation code that is associated with a "Markham" and I don't see any way to use my own Copilot subscription.

Anyone know what the story is?
@BeckerFrank @merks

**Comment by BeckerFrank** [comment]
Maybe this is disabled / not possible because the organization is in the Enterprise eclipse-foundation.
Did you ask the Helpdesk?

**Comment by gnl42** [comment]
My initial assumption was that it was part of the Oomph setup, but when I try to set it up on my own Eclipse instance I see the same issue. I'll raise an issue with GitHub and hopefully they can explain what it going on
